github/orgs_teams.go

```diff
@@ -46,17 +46,46 @@ func (t Team) String() string {
 
 // Invitation represents a team member's inviation status
 type Invitation struct {
-	ID        *int       `json:"id,omitempty"`
-	Login     *string    `json:"login,omitempty"`
-	Email     *string    `json:"email,omitempty"`
-	Role      *string    `json:"role,omitempty"`
-	CreatedAt *time.Time `json:"created_at,omitempty"`
+	ID    *int    `json:"id,omitempty"`
+	Login *string `json:"login,omitempty"`
+	Email *string `json:"email,omitempty"`
+	Role  *string `json:"role,omitempty"`
+	//golang documentation states
+	//Programs using times should typically store and pass them as values, not pointers.
+	//That is, time variables and struct fields should be of type time.Time, not *time.Time
```

Comment:
This comment shouldn't be here as part of the final PR.

I will respond to it.

While it's true that it's fine to use `time.Time` as a value, the reason we're using it as a point here is because of JSON encoding. We want to be able to tell (sometimes) that a field is absent by checking if its value is nil.

So, we want to keep `CreatedAt` as a pointer to `*time.Time`. If you want to discuss this further, I suggest opening a new issue, because it's outside of the scope of this PR.